viewer/packages/webpack.config.js

```diff
@@ -31,9 +36,18 @@ module.exports = env => {
           use: {
             loader: 'ts-loader',
             options: {
-              configFile: path.resolve(__dirname, '../tsconfig.json')
+              configFile: path.resolve(__dirname, '../tsconfig.json'),
+              compilerOptions: {
+                noUnusedLocals: false,
+                noUnusedParameters: false
+              }
```

Comment:
Why do we need to change this?

Reply:
Since this is the webpack config for the test app, it is VERY annoying that you are unable to run your app if you are in the middle of creating some feature and not everything is in use (yet).